packages/main/src/RPA/Excel/Files.py

```diff
@@ -787,6 +787,14 @@ def append_worksheet(self, name=None, content=None, header=False, start=None):
 
     def remove_worksheet(self, name=None):
         name = self._get_sheetname(name)
+        others = [sheet for sheet in self.sheetnames if sheet != name]
+
+        if not others:
+            raise ValueError("Workbook must have at least one worksheet")
```

Comment:
Nit
```suggestion
            raise ValueError("Workbook must have at least one other worksheet")
```
if that makes more sense, so the user understands that his one and only worksheet isn't enough as it leaves the Excel empty.

Reply:
Fixed in both places